### PR TITLE
Various edits in "Proof-of-Stake" webpage

### PR DESCRIPTION
**Pull request by qe**
## Description

Various content edits in **PROOF-OF-STAKE (POS)** webpage (spelling, grammar, flow)

## Related Issue

None

### CODE REVIEW

File: src/content/developers/docs/consensus-mechanisms/pos/index.md [hunk 1]
```diff
@@ -27,37 +27,37 @@ Proof-of-stake comes with a number of improvements to the proof-of-work system:
...
 
 If you attest to malicious blocks, you lose your stake.
 
 ### The beacon chain {#the-beacon-chain}
 
 When Ethereum replaces proof-of-work with proof-of-stake, there will be the added complexity of [shard chains](/eth2/shard-chains/). These are separate blockchains that will need validators to process transactions and create new blocks. The plan is to have 64 shard chains and they all need a shared understanding of the state of the network. So extra coordination is needed and this will be done by [the beacon chain](/eth2/beacon-chain/).
 
-The beacon chain receives state information from shards and makes it available for other shards so that the network can stay in sync. The beacon chain will also manage the validators, from registering their stake deposits to issuing their rewards and penalties.
+The beacon chain receives state information from shards and makes it available for other shards so that the network can stay in sync. From registering their stake deposits to issuing their rewards and penalties, the beacon chain will also manage the validators.
```
Comment (wackerow) [inline src/content/developers/docs/consensus-mechanisms/pos/index.md:42]: Only change I'm not so sure on... feel the sentence structure just reads a little more awkwardly after flipping, could be convinced though, thoughts?

**Review by wackerow — CHANGES_REQUESTED** [review]
@qe Thanks for doing this =) Just made one comment, let me know your thoughts and we'll get this in

File: src/content/developers/docs/consensus-mechanisms/pos/index.md (hunk 1 above)
Comment (qe) [reply src/content/developers/docs/consensus-mechanisms/pos/index.md]: @wackerow 
For sure! This weekend, I'll look through the rest of the pages for more potential edits to help out with the content. I see what you're referring to with ***The beacon chain*** section's flow, so I just reverted the original sentence in my most recent commit. I also added a few more edits that I noticed when rereading the doc.

File: src/content/developers/docs/consensus-mechanisms/pos/index.md [hunk 2]
```diff
@@ -27,37 +27,37 @@ Proof-of-stake comes with a number of improvements to the proof-of-work system:
...
+If a validator isn't chosen to propose a new shard block, they'll have to attest to another validator's proposal and confirm that everything looks as it should. It's the attestation that is recorded in the beacon chain rather than the transaction itself.
 
-At least 128 validators are required to attest to each shard block – this is known as a "committee".
+At least 128 validators are required to attest to each shard block – this is known as a "committee."
 
-The committee has a time-frame in which to propose and validate a shard block. This is known as a "slot". Only one valid block is created per slot. There are 32 slots in an "epoch". After each epoch, the committee is disbanded and reformed with different, random participants. This helps keep shards safe from committees of bad actors.
+The committee has a time-frame in which to propose and validate a shard block. This is known as a "slot." Only 1 valid block is created per slot, and there are 32 slots in an "epoch." After each epoch, the committee is disbanded and reformed with different, random participants. This helps keep shards safe from committees of bad actors.
```
Comment (wackerow) [inline src/content/developers/docs/consensus-mechanisms/pos/index.md:56]: Would keep numbers under 10 spelled out

File: src/content/developers/docs/consensus-mechanisms/pos/index.md [hunk 3]
```diff
@@ -71,16 +71,16 @@ As Vlad Zamfir put it, this is like a miner participating in a 51% attack, causi
...
+The threat of a [51% attack](https://www.investopedia.com/terms/1/51-attack.asp) still exists in proof-of-stake, but it's even more risky for the attackers. To do so, you'd need to control 51% of the staked ETH. Not only is this a lot of money, but it would probably cause ETH's value to drop. There's very little incentive to destroy the value of a currency you have a majority stake in. There are stronger incentives to keep the network secure and healthy.
 
-Stake slashings, ejections, and other penalties, coordinated by the beacon chain, will exist to prevent other acts of bad behaviour. Validators will also be responsible for flagging these incidents.
+Stake slashings, ejections, and other penalties, coordinated by the beacon chain, will exist to prevent other acts of bad behavior. Validators will also be responsible for flagging these incidents.
```
Comment (wackerow) [inline src/content/developers/docs/consensus-mechanisms/pos/index.md:76]: https://github.com/ethereum/ethereum-org-website/issues/2786
Hah, note there is an open issue about British/American English inconsistencies... It's fine here, but before going much further with this specific type of change, let's see what Ryan thinks

**Review by wackerow — CHANGES_REQUESTED** [review]
@qe Nice, real close, one last nit-picky request, sorry =) Then I'll pull it in

File: src/content/developers/docs/consensus-mechanisms/pos/index.md [hunk 4]
```diff
@@ -27,37 +27,37 @@ Proof-of-stake comes with a number of improvements to the proof-of-work system:
...
 
 ## How does Ethereum's proof-of-stake work? {#how-does-pos-work}
 
-Unlike proof-of-work, validators don't need to use significant amounts of computational power because they're selected at random and aren't competing. They don't need to mine blocks, they just need to create blocks when chosen and validate proposed blocks when they're not. This validation is known as attesting. You can think of attesting as saying "this block looks good to me". Validators get rewards for proposing new blocks and for attesting to ones they've seen.
+Unlike proof-of-work, validators don't need to use significant amounts of computational power because they're selected at random and aren't competing. They don't need to mine blocks; they just need to create blocks when chosen and validate proposed blocks when they're not. This validation is known as attesting. You can think of attesting as saying "this block looks good to me." Validators get rewards for proposing new blocks and for attesting to ones they've seen.
```
Comment (minimalsm) [inline src/content/developers/docs/consensus-mechanisms/pos/index.md:34]: > You can think of attesting as saying "this block looks good to me." Validators get rewards for proposing new blocks and for attesting to ones they've seen.

Putting the period inside the quotation marks turns this into one sentence.

**Review by minimalsm — CHANGES_REQUESTED** [review]
Just a small thing I noticed

File: src/content/developers/docs/consensus-mechanisms/pos/index.md (hunk 2 above)
Comment (qe) [reply src/content/developers/docs/consensus-mechanisms/pos/index.md]: Just fixed it!

**Review by wackerow — APPROVED** [review]
LGTM, thanks!

**Comment by wackerow** [comment]
@all-contributors please add @qe for content

**Pull request merged**